CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#82](https://github.com/nf-core/nanostring/pull/82) - Updated to nf-core template 3.0.2
 - [#95](https://github.com/nf-core/nanostring/pull/95) - Add pipeline level nf-tests.
 - [#96](https://github.com/nf-core/nanostring/pull/96) - Remove timestamp suffix from nacho_norm.R and write_out_prepared_gex.R
+- [#99](https://github.com/nf-core/nanostring/pull/99) - Add nf-test for local module nacho_normalize.
```

Comment:
Doesn't have any impact at all, but @atrigila pointed out to me that we should add the process names the exact same way as they appear defined in the module and between ``, so that it looks nicer when the .md document is rendered. In this case should be `NACHO_NORMALIZE`